analyses/cnv-chrom-plot/util/bin-coverage.R

```diff
@@ -0,0 +1,142 @@
+# Functions for calling CN statuses of genome bins
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2020
+
+bp_per_bin <- function(bin_ranges, status_ranges) {
+  # Given a binned genome ranges object and another GenomicRanges object, 
```

Comment:
Yes, in that, that is what it is being used for but technically it would work for any range you gave it, so I didn't specify that it had to be concerned with CN status.

Reply:
Ahh okay, I understand.

Perhaps instead of specifying that this other GenomicRanges object here, you can specify in the module's `README` file where you mention this function that the other GenomicRanges object in the case of this analysis, contains the copy number status ranges.

This decision is however completely up you. Like I said, I believe the steps in the function are clearly broken up enough to be followed by someone who did not write them, but added clarification in this case won't hurt (you can also argue that it won't make a significant difference so again up to you).

Reply:
I'll make a clarification in the README!